test/core/storage/trie/polkadot_trie/polkadot_trie_cursor_test.cpp

```diff
@@ -10,20 +10,22 @@
 #include <gtest/gtest.h>
 
 #include "storage/trie/polkadot_trie/polkadot_trie_impl.hpp"
+#include "testutil/storage/polkadot_trie_printer.hpp"
 #include "testutil/literals.hpp"
 #include "testutil/outcome.hpp"
 
 using kagome::common::Buffer;
 using kagome::storage::trie::PolkadotTrie;
 using kagome::storage::trie::PolkadotTrieCursor;
 using kagome::storage::trie::PolkadotTrieImpl;
+using kagome::storage::trie::operator<<;
 
-class PolkadotTrieCursorTest : public testing::Test {
-};
+class PolkadotTrieCursorTest : public testing::Test {};
 
-std::tuple<std::unique_ptr<PolkadotTrie>, std::set<Buffer>> generateRandomTrie(size_t keys_num,
-                        size_t max_key_length = 32,
-                        size_t key_alphabet_size = 16) noexcept {
+std::tuple<std::unique_ptr<PolkadotTrie>, std::set<Buffer>> generateRandomTrie(
+    size_t keys_num,
+    size_t max_key_length = 32,
```

Comment:
declare constants beforehand and use them here, those values are not obvious

Reply:
Eh, they're kinda pulled out of a hat. Okay, I will mention it.